string base64
string content_type
int32 height
string name
bool system_asset
int32 width